Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/BoundsTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class BoundsTests : TypeTester<Bounds>
+    {
+        public static (Bounds deserialized, string serialized)[] representations = new[] {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use an immutable collection or reduce the accessibility of the field(s) 'representations'.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)